09-workload.md

```diff
@@ -20,6 +20,49 @@ The cluster now has an [Traefik configured with a TLS certificate](./08-secret-m
    kubectl wait -n a0008 --for=condition=ready pod --selector=app.kubernetes.io/name=aspnetapp --timeout=90s
    ```
 
+1. Deploy the Ingress resource
+
+   ```bash
+   cat <<EOF | kubectl create -f -
+   apiVersion: networking.k8s.io/v1
+   kind: Ingress
+   metadata:
+     name: aspnetapp-ingress
+     namespace: a0008
+     annotations:
+       kubernetes.io/ingress.allow-http: "false"
+       # defines controller implementing this ingress resource: https://docs.microsoft.com/en-us/azure/dev-spaces/how-to/ingress-https-traefik
+       # ingress.class annotation is being deprecated in Kubernetes 1.18: https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation
+       # For backwards compatibility, when this annotation is set, precedence is given over the new field ingressClassName under spec.
+       kubernetes.io/ingress.class: traefik-internal
+       traefik.ingress.kubernetes.io/router.entrypoints: websecure
+       traefik.ingress.kubernetes.io/router.tls: "true"
+       traefik.ingress.kubernetes.io/router.tls.options: default
+       traefik.ingress.kubernetes.io/router.middlewares: app-gateway-snet@file, gzip-compress@file
+   spec:
+     # ingressClassName: "traefik-internal"
+     tls:
+     - hosts:
+         - bu0001a0008-00.aks-ingress.$DOMAIN_NAME
+           # it is possible to opt for certificate management strategy with dedicated
+           # certificates for each TLS SNI route.
+           # In this Rereference Implementation for the sake of simplicity we use a
+           # wildcard default certificate added at Ingress Controller configuration level which is *.example.com
+           # secretName: <bu0001a0008-00-example-com-tls-secret>
```

Comment:
This is set up as the default in Traefik's config, https://github.com/mspnp/aks-secure-baseline/blob/b3319376ac58fe28ad5244be2776bbf84633b47a/workload/traefik.yaml#L295 -- which has the other name hardcoded in there.  I'm thinking that secret name (KV and CSI KV Driver reference) should be genericized as well as part of this PR.

Reply:
Looks like you've got them all updated, nice!